src/Rings/affine-algebra-homs.jl

```diff
@@ -243,7 +243,7 @@ function (F::AlgHom)(p::U) where U <: Union{MPolyElem, MPolyQuoElem}
 end
 
 @doc Markdown.doc"""
-    function domain(F::AlgHom)
+    domain(F::AlgHom)
```

Comment:
I don't think the point of a doc string is to prove that the function exists. If this is the case, I see no need whatsoever for doc string attached to `f(x)` that says "return f of x".

Reply:
Sure. But I did not review which docstrings should go (many should) or should be added, this was entirely about fixing an obvious bug in multiple docstrings via a regex.

Reply:
Ok. Glad to know we still should not be writing doc strings like this.

Reply:
I removed these and two other redundant docstrings for `domain` and `codomain`. Plenty more redundant docstrings remain, but I don't want to get lost in chasing those down right now.